packages/file-search/src/common/file-search-service.ts

```diff
@@ -33,13 +33,24 @@ export interface FileSearchService {
 
 export const FileSearchService = Symbol('FileSearchService');
 export namespace FileSearchService {
-    export interface Options {
-        rootUris: string[]
+    export interface BaseOptions {
+        useGitIgnore?: boolean
```

Comment:
just wondering: why is gitignore more special than the others?

Reply:
I have not thought about semantics fo `useGitIgnore` (only about globs) in this PR and moved it here to be able to pass it tor https://github.com/theia-ide/theia/pull/4602/files/b8c3cff127161cc5509089b0f1bbb3f7ef5196ce#diff-6ba24f8b01d65a9959cba7e78f9a050bR131

Would it be alright to rethink it separately? Could you open an issue?


Renaming it would be a breaking change,

Reply:
yea for sure it doesn't belong to this PR even if other .ignore files need to be supported.